pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -278,6 +278,7 @@ private static long getRandomInitialDelayInSeconds() {
   private static final String SERVER_ADMIN_REQUEST_TIMEOUT_SECONDS = "server.request.timeoutSeconds";
   private static final String MINION_ADMIN_REQUEST_TIMEOUT_SECONDS = "minion.request.timeoutSeconds";
   private static final String SEGMENT_COMMIT_TIMEOUT_SECONDS = "controller.realtime.segment.commit.timeoutSeconds";
+  private static final String CONTROLLER_NUM_THREAD_POOL = "controller.numThreadPool";
```

Comment:
`numThreadPool` is a little bit misleading, how about `controller.executor.numThreads`